src/toil/__init__.py

```diff
@@ -144,7 +144,7 @@ def resolveEntryPoint(entryPoint):
 
 
 @memoize
-def physicalMemory():
+def physicalMemory() -> int:
```

Comment:
Do you want to expand the mypy CI checks?

Reply:
Good point!  I will add it in a separate PR.